sass/compat/themes/minimog/_checkout-payment-methods.scss

```diff
@@ -0,0 +1,7 @@
+// ------------------------------------
+// CHECKOUT PAYMENT METHOD
+// ------------------------------------
+
+body#{ $_body-theme-selector } .wc_payment_methods li + li{
+    margin: 0 !important; //original property from the theme css uses !important
```

Comment:
This is ok, but for consistency, use the comment `// Needed to override theme styles` for these cases.